certs/pkiutil/pki_helpers.go

```diff
@@ -34,12 +34,13 @@ import (
 	"path/filepath"
 	"time"
 
```

Comment:
Thanks for doing this .. as a nit though, we should probably combine this into the block above (with other "stdlib" imports)